server/sonar-web/src/main/js/apps/quality-gates/components/AddConditionForm.js

```diff
@@ -30,15 +30,15 @@ export default function AddConditionForm({ metrics, onSelect }) {
     onSelect(metric);
   }
 
-  const metricsToDisplay = metrics.filter(metric => !metric.hidden);
-  const sortedMetrics = sortBy(metricsToDisplay, 'domain');
-  const options = sortedMetrics.map(metric => {
-    return {
+  const metricsToDisplay = omitBy(metrics, metric => metric.hidden);
```

Comment:
it's a matter of taste, but I prefer using native array `filter`

Reply:
I did it like this because `metrics` is an object with a key for every metric, not an array.